core/src/tile/tileWorker.h

```diff
@@ -32,6 +34,8 @@ class TileWorker : public TileTaskQueue {
 
     void setScene(std::shared_ptr<Scene>& _scene);
 
+    std::shared_ptr<const Platform> getPlatform() { return m_platform; }
```

Comment:
It's used in the platform code in general, but I would prefer not to expose that :/
Let me know if you have ideas on how we could get rid of this.